packages/wallets/bitkeep/src/adapter.ts

```diff
@@ -1,3 +1,5 @@
+import { WalletInfo, SolanaAdapter } from '@bitget-wallet/web3-sdk';
+import type { SolanaProvider } from '@bitget-wallet/web3-sdk';
```

Comment:
Please import only as types here. The actual library will need to be imported async (see comment below).